home_assistant_streamdeck_yaml.py

```diff
@@ -1452,6 +1842,62 @@ def _generate_failed_icon(
     )
     return icon
 
+def update_all_dials(
+    deck: StreamDeck,
+    config: Config,
+    complete_state: StateDict,
+):
+    console.log("Called update_all_dials")
+    for key in range(len(config.current_page().dials)):
+        current_dial = config.current_page().dials[key]
+        assert current_dial is not None
+        if current_dial.entity_id is None:
+            return
+        update_dial(deck, key, config, complete_state, complete_state[current_dial.entity_id])
+    
+    
+def update_dial(
+    deck: StreamDeck,
+    key: int,
+    config: Config,
+    complete_state: StateDict,
+    data: dict[str, Any] | None = None,
+) -> None:
+    """Update the dial"""
+    dial = config.dial(key)
+    assert dial is not None
+    
+    if dial.dial_event_type == "PUSH": 
+        return
+    
+    if data is not None:
+        try:
```

Comment:
please, try to avoid heavy try/except usage. there are much better ways to solve this